config/defines.inc.php

```diff
@@ -69,7 +69,14 @@
     define('_PS_BO_ALL_THEMES_DIR_', _PS_ADMIN_DIR_.'/themes/');
 }
 if (!defined('_PS_CACHE_DIR_')) {
-    $prestashopCacheDir =  _PS_ROOT_DIR_.'/var/cache/'.(_PS_MODE_DEV_ ? 'dev': 'prod'). DIRECTORY_SEPARATOR;
```

Comment:
Currently refactoring this one in https://github.com/PrestaShop/PrestaShop/pull/14894